src/main/java/com/capitalone/dashboard/collector/GitHubCollectorTask.java

```diff
@@ -166,84 +166,103 @@ public void collect(Collector collector) {
 
         clean(collector);
         List<GitHubRepo> enabledRepos = enabledRepos(collector);
+        LOG.info("GitHubCollectorTask:collect start, total enabledRepos=" + enabledRepos.size());
         for (GitHubRepo repo : enabledRepos) {
```

Comment:
please ensure the enabledRepos does not contain nulls by filtering the nulls before iterating on it.

Reply:
added null checks in appropriate places.

Reply:
Thanks